src/kafka-net/BrokerRouter.cs

```diff
@@ -98,18 +104,14 @@ public BrokerRoute SelectBrokerRoute(string topic, byte[] key = null)
         /// The topic metadata will by default check the cache first and then if it does not exist it will then
         /// request metadata from the server.  To force querying the metadata from the server use <see cref="RefreshTopicMetadata"/>
         /// </remarks>
-        public List<Topic> GetTopicMetadata(params string[] topics)
+        /// <exception cref="InvalidTopicNotExistsInCache">Condition: not exists in Cache</exception>
```

Comment:
Why the "Condition:"?? just write "Thrown if the topic metadata does not exist in the cache" (notice no capital letter in cache)